### PR TITLE
add config description for opentracing

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
add config description for opentracing

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7345
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by tangenta — APPROVED** [review]

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -488,6 +488,78 @@ The [`plan cache`](/sql-prepared-plan-cache.md) configuration of the `PREPARE` s
...
++ Default value: `""`
+
+### `max-operations`
+
++ The maximum number of operations that the sampler can trace. If an operation is not traced, the default probabilistic sampler is used.
++ Default value: `0`
+
+### `sampling-refresh-interval`
+
++ Controls the frequency of polling the jaeger-agent sampling policy.
```
Comment (en-jin19) [inline tidb-configuration-file.md:536]:
```suggestion
+ Controls the frequency of remote polling the jaeger-agent sampling policy.
```

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -488,6 +488,78 @@ The [`plan cache`](/sql-prepared-plan-cache.md) configuration of the `PREPARE` s
...
+Configuration items related to opentracing.reporter.
+
+### `queue-size`
+
++ The queue size with which the reporter records spans in memory.
++ Default value: `0`
+
+### `buffer-flush-interval`
+
++ The interval at which the reporter flushes the spans in memory to the storage.
```
Comment (en-jin19) [inline tidb-configuration-file.md:550]: I think this description provides more details than the Chinese one. If possible, could we update the Chinese description later according to the English one?

**Review by en-jin19 — APPROVED** [review]
rest LGTM

**Pull request merged**